test_util/test_runner.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+"""Module for running integration_test.py inside of a remote cluster
```

Comment:
Note for this, we don't have SSH in a bunch of our postflights. AWS, Azure don't have SSH, but ideally they all should work about hte same way.